code/game/turfs/simulated/floor/plating/lava.dm

```diff
@@ -54,10 +54,14 @@
 				O.armor["fire"] = 50
 			O.fire_act(10000, 1000)
 
-
 		else if (isliving(thing))
 			. = 1
 			var/mob/living/L = thing
+			if(ishuman(thing))
```

Comment:
I'm guessing it's because I asked him to not give angel wings lava immunity. However, that reference has grown old now, so @kevinz000  should probably just switch it to the movement type in general instead of the specific exception.